cmd/svcat/testdata/output/get-plans-by-class.txt

```diff
@@ -1,4 +1,9 @@
-   NAME     NAMESPACE           CLASS                 DESCRIPTION        
-+---------+-----------+-----------------------+-------------------------+
-  default               user-provided-service   Sample plan description  
-  premium               user-provided-service   Premium plan             
+   NAME     NAMESPACE            CLASS                      DESCRIPTION            
++---------+-----------+--------------------------+--------------------------------+
+  default               user-provided-service      Sample plan description         
+  premium               user-provided-service      Premium plan                    
+  default               another-provided-service   Another sample plan             
+                                                   description that's really       
```

Comment:
where did the extra test data come from? 🤔

Reply:
Renamed a field incorrectly by accident D:

Fixed this now. Thanks for the catch!